lib/Middleware/ListenersMiddleware.php

```diff
@@ -38,6 +38,8 @@
 use Xibo\Event\SystemUserChangedEvent;
 use Xibo\Event\UserDeleteEvent;
 use Xibo\Listener\CampaignListener;
+use Xibo\Event\ConnectorReportEvent;
```

Comment:
These can be removed now I think?

Reply:
ConnectorReportEvent is needed as it sets the connector reports object

Reply:
But it isn't used in this middleware?